Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1430,50 +1433,25 @@ void CGameListCtrl::AutomaticColumnWidth()
   }
   else if (GetColumnCount() > 0)
   {
-    int resizable =
-        rc.GetWidth() - (GetColumnWidth(COLUMN_PLATFORM) + GetColumnWidth(COLUMN_BANNER) +
-                         GetColumnWidth(COLUMN_ID) + GetColumnWidth(COLUMN_COUNTRY) +
-                         GetColumnWidth(COLUMN_SIZE) + GetColumnWidth(COLUMN_EMULATION_STATE));
-    if (SConfig::GetInstance().m_showTitleColumn && SConfig::GetInstance().m_showMakerColumn &&
-        SConfig::GetInstance().m_showFileNameColumn)
-    {
-      SetColumnWidth(COLUMN_TITLE, resizable / 3);
-      SetColumnWidth(COLUMN_MAKER, resizable / 3);
-      SetColumnWidth(COLUMN_FILENAME, resizable / 3);
-    }
-    else if (SConfig::GetInstance().m_showMakerColumn &&
-             SConfig::GetInstance().m_showFileNameColumn)
-    {
-      SetColumnWidth(COLUMN_MAKER, resizable / 2);
-      SetColumnWidth(COLUMN_FILENAME, resizable / 2);
-    }
-    else if (SConfig::GetInstance().m_showMakerColumn && SConfig::GetInstance().m_showTitleColumn)
-    {
-      SetColumnWidth(COLUMN_MAKER, resizable / 2);
-      SetColumnWidth(COLUMN_TITLE, resizable / 2);
-    }
-    else if (SConfig::GetInstance().m_showFileNameColumn &&
-             SConfig::GetInstance().m_showTitleColumn)
-    {
-      SetColumnWidth(COLUMN_TITLE, resizable / 2);
-      SetColumnWidth(COLUMN_FILENAME, resizable / 2);
-    }
-    else if (SConfig::GetInstance().m_showMakerColumn)
-    {
-      SetColumnWidth(COLUMN_MAKER, resizable);
-    }
-    else if (SConfig::GetInstance().m_showFileNameColumn)
-    {
-      SetColumnWidth(COLUMN_FILENAME, resizable);
-    }
-    else if (SConfig::GetInstance().m_showTitleColumn)
-    {
-      SetColumnWidth(COLUMN_TITLE, resizable);
-    }
-    else
+    int resizable = rc.GetWidth();
```

Comment:
It may be better to rename this. This is a name more ideal for a boolean variable, but not really one suited for one that has arithmetic performed on it. i.e. its current name doesn't really make it's purpose clear.